src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1018,7 +1019,7 @@ TEST(SpatialPoolerTest, testUpdateBoostFactors) {
   sp.updateBoostFactors_();
   sp.getBoostFactors(resultBoostFactors4.data());
 
-  ASSERT_TRUE(check_vector_eq(trueBoostFactors3, resultBoostFactors3));
+  ASSERT_TRUE(check_vector_eq(trueBoostFactors4, resultBoostFactors4));
```

Comment:
has this always been causing problems? ie unrelated to the logic changes in this PR?